outputs_vnet.tf

```diff
@@ -0,0 +1,36 @@
+output "name" {
```

Comment:
Can you prefix each output with the resource type? for example `vnet_`

Reply:
updated